EMML/1001-2000/EMML1292.xml

```diff
@@ -32,40 +32,981 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0004HMML"/>
-                  <collection>Addis Ababa University. YaʼItyop̣yā ṭenātenā meremer taqwām, Addis Ababa</collection>
+                  <collection>Addis Ababa University.
```

Comment:
please change repository and collection as here in Example 2 
https://betamasaheft.eu/Guidelines/?q=EMML&id=identifiersStructure

Reply:
ok.